### PR TITLE
add 'code' flag, fix 'path' to use directories

### PR DESCRIPTION
**Pull request by DonKeyHot1**

### CODE REVIEW

File: bin/acquit-markdown.js [hunk 1]
```diff
@@ -27,8 +28,31 @@ if (commander.require) {
...
+  };
+
+  getFiles(commander.path).forEach(file => {
+    markdown += acquit.parse(fs.readFileSync(file, 'utf8'));
+  });
+} else {
+  markdown = acquit.parse(fs.readFileSync(commander.path, 'utf8'));
+}
+
+markdown = markdown.replace(/eslint-\S*/g, '').replace(/TODO[\s\S]*$/g, '');
```
Comment (vkarpov15) [inline bin/acquit-markdown.js]: I'm a little concerned about this line. Is it necessary? Stripping out 'TODO' seems like it's outside the scope of this tool

Comment (DonKeyHot1) [reply bin/acquit-markdown.js]: I've used this tool to generate documentation for my tests and I was a little confused about TODO's in the middle of the generated text. So in my opinion there are could be few TODO's in your tests, but you don't want to see them in documentation

Comment (vkarpov15) [reply bin/acquit-markdown.js]: Can we put it behind a flag? I'd rather not make this a default behavior - tricky to document. Maybe a `--replace` flag that takes in a regexp?
